pennylane/transforms/metric_tensor.py

```diff
@@ -1,641 +1,641 @@
-
-        # Prepare the mask to match the used interface
-        mask = qml.math.convert_like(mask, diag_mt)
-
-        # Initialize off block-diagonal tensor using the stored ids
-        first_term = qml.math.zeros_like(diag_mt)
-        if ids:
-            off_diag_res = qml.math.stack(off_diag_res, 1)[0]
-            inv_ids = [_id[::-1] for _id in ids]
-            first_term = qml.math.scatter_element_add(first_term, list(zip(*ids)), off_diag_res)
-            first_term = qml.math.scatter_element_add(first_term, list(zip(*inv_ids)), off_diag_res)
-
-        # Second terms of off block-diagonal metric tensor
-        expvals = qml.math.zeros_like(first_term[0])
-        idx = 0
-        for prob, obs in zip(diag_res, obs_list):
-            for o in obs:
-                l = qml.math.cast(o.eigvals, dtype=np.float64)
-                w = tape.wires.indices(o.wires)
-                p = qml.math.marginal_prob(prob, w)
-                expvals = qml.math.scatter_element_add(expvals, (idx,), qml.math.dot(l, p))
-                idx += 1
-
-        # Construct <\partial_i\psi|\psi><\psi|\partial_j\psi> and mask it
-        second_term = qml.math.tensordot(expvals, expvals, axes=0) * mask
-
-        # Subtract second term from first term
-        off_diag_mt = first_term - second_term
-
-        # Rescale first and second term
-        _coeffs = qml.math.hstack(coeffs)
-        scale = qml.math.convert_like(qml.math.tensordot(_coeffs, _coeffs, axes=0), results[0])
-        off_diag_mt = scale * off_diag_mt
-
-        # Combine block-diagonal and off block-diagonal
-        mt = off_diag_mt + diag_mt
-
-        return mt
-
-    return tapes, processing_fn
-
-
-def _get_aux_wire(aux_wire, tape, device_wires):
-    r"""Determine an unused wire to be used as auxiliary wire for Hadamard tests.
-
-    Args:
-        aux_wire (object): Input auxiliary wire. Returned unmodified if not ``None``
-        tape (pennylane.tape.QuantumTape): Tape to infer the wire for
-        device_wires (.wires.Wires): Wires of the device that is going to be used for the
-            metric tensor. Facilitates finding a default for ``aux_wire`` if ``aux_wire``
-            is ``None`` .
-
-    Returns:
-        object: The auxiliary wire to be used. Equals ``aux_wire`` if it was not ``None`` ,
-        and an often reasonable choice else.
-    """
-    if aux_wire is not None:
-        if device_wires is None or aux_wire in device_wires:
-            return aux_wire
-
-    if device_wires is not None:
-        unused_wires = qml.wires.Wires(device_wires.toset().difference(tape.wires.toset()))
-        if not unused_wires:
-            raise qml.wires.WireError("No device wires are unused by the tape.")
-        return unused_wires[0]
-
-    _wires = tape.wires
-    for _aux in range(tape.num_wires):
-        if _aux not in _wires:
-            return _aux
-
-    return tape.num_wires
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
For some reason this entire file looks different? Has anything changed here?

Reply:
Oh no, this must be a rendering issue, I should only have updated names (if at all)